docs/core/setup/install-runtime.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Install .NET Core runtime on Windows, Linux, and macOS - .NET Core
+description: Learn how to install .NET Core on Windows, Linux, and macOS. Discover the dependencies required to run .NET Core apps.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+ms.custom: "updateeachrelease"
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Install the .NET Core Runtime
```

Comment:
```suggestion
# Install the .NET Core runtime
```

Reply:
@Thraka and I discussed this and I was going to go back to the team on this. I had the same suggestion and deleted.

When referring to .NET Core Runtime as the product, we'd use uppercase. Thoughts?

Reply:
I added a task to discuss and update these articles with runtime or Runtime, once we figure it out. I'll ping an editor too.